spec/stat_tracker_spec.rb

```diff
@@ -73,37 +73,53 @@
 
   describe 'GameStats module methods' do
 
-      it "highest_total_score" do
+    describe 'highest_total_score' do
```

Comment:
Good use of spacing and describe blocks!

Reply:
The extra describe blocks really clean it up nice.